monai/networks/utils.py

```diff
@@ -46,12 +46,19 @@ def slice_channels(tensor, *slicevals):
     return tensor[slices]
 
 
-def predict_segmentation(logits):
+def predict_segmentation(logits, mutually_exclusive=False, threshold=0):
     """
-    Given the logits from a network, computing the segmentation by thresholding all values above 0 if `logits` has one
-    channel, or computing the `argmax` along the channel axis otherwise, logits has shape `BCHW[D]`
+    Given the logits from a network, computing the segmentation by thresholding all values above 0
+    if multi-labels task, computing the `argmax` along the channel axis if multi-classes task,
+    logits has shape `BCHW[D]`.
+
+    Args:
+        logits (Tensor): raw data of model output.
+        mutually_exclusive (Bool): if True, `logits` will be converted into a binary matrix using
+            a combination of argmax, which is suitable for multi-classes task. Defaults to False.
+        threshold (float): thresholding the prediction values if multi-labels task.
     """
-    if logits.shape[1] == 1:
```

Comment:
warns if channel==1 and mutuall_exclusive

Reply:
updated in the latest patch.
Thanks.